Formula/cairo.rb

```diff
@@ -47,6 +54,10 @@ def install
       args << "--enable-xcb=no" << "--enable-xlib=no" << "--enable-xlib-xrender=no"
     end
 
+    if build.head?
+      system "./autogen.sh", *args
+    end
+
```

Comment:
Let's make this `system "./autogen.sh", *args if build.head?` and put this directly in front of the other `system` calls for brevity.